vaadin-text-field-flow-parent/vaadin-text-field-flow-integration-tests/src/test/java/com/vaadin/flow/component/textfield/tests/TextAreaPatternIT.java

```diff
@@ -1,129 +0,0 @@
-/*
- * Copyright 2000-2022 Vaadin Ltd.
- *
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not
- * use this file except in compliance with the License. You may obtain a copy of
- * the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
- * License for the specific language governing permissions and limitations under
- * the License.
- */
-package com.vaadin.flow.component.textfield.tests;
-
-import com.vaadin.flow.component.textfield.TextArea;
-import com.vaadin.flow.component.textfield.testbench.TextAreaElement;
-import com.vaadin.flow.testutil.TestPath;
-import com.vaadin.testbench.TestBenchElement;
-import com.vaadin.tests.AbstractComponentIT;
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.Ignore;
-import org.junit.Test;
-
-/**
- * Integration tests for {@link TextArea}.
- */
-@TestPath("vaadin-text-field/text-area-pattern")
-public class TextAreaPatternIT extends AbstractComponentIT {
-
-    private TextAreaElement textArea;
-    private TestBenchElement currentValue;
-    private TestBenchElement setNumberPattern;
-    private TestBenchElement setInvalidPattern;
-    private TestBenchElement clearPattern;
-    private TestBenchElement checkValidity;
-    private TestBenchElement validityOutput;
-    private TestBenchElement enableAllowedCharPattern;
-
-    @Before
-    public void init() {
-        open();
-        textArea = $(TextAreaElement.class).first();
-        currentValue = $("div").id("current-value");
-        setNumberPattern = $("button").id("set-number-pattern");
-        setInvalidPattern = $("button").id("set-invalid-pattern");
-        clearPattern = $("button").id("clear-pattern");
-        enableAllowedCharPattern = $("button")
-                .id("enable-allowed-char-pattern");
-        // These two are defined by TextFieldTestPageUtil.addInvalidCheck
-        checkValidity = $("button").id("check-is-invalid");
-        validityOutput = $("div").id("is-invalid");
-    }
-
-    private void assertClientSideValid() {
-        Assert.assertFalse(textArea.hasAttribute("invalid"));
-    }
-
-    private void assertClientSideInvalid() {
-        Assert.assertTrue(textArea.hasAttribute("invalid"));
-    }
-
-    private void assertServerSideValid() {
-        checkValidity.click();
-        Assert.assertEquals("valid", validityOutput.getText());
-    }
-
-    private void assertServerSideInvalid() {
-        checkValidity.click();
-        Assert.assertEquals("invalid", validityOutput.getText());
-    }
-
-    @Test
-    public void validPattern_validInput_isValid() {
-        setNumberPattern.click();
-        textArea.sendKeys("1234");
-        blur();
-
-        assertClientSideValid();
-        assertServerSideValid();
-    }
```

Comment:
**note:** Already covered by this tests:

https://github.com/vaadin/flow-components/blob/4477ce194a7491f8e7777f40f3e0162d421d2d90/vaadin-text-field-flow-parent/vaadin-text-field-flow-integration-tests/src/test/java/com/vaadin/flow/component/textfield/tests/validation/TextAreaBasicValidationIT.java#L120-L130